resource/css/styles.css

```diff
@@ -953,10 +968,13 @@ div.input-group > .input-group-addon {
   min-width: 65px;
 }
 
-.alphabetical-search-results li, .group-container > p, .group-container > ul {
+.alphabetical-search-results li,
+.group-container > p,
+.group-container > ul {
   font-size: 14px;
   line-height: 15px;
   list-style-type: none;
+  line-height: 2em;
```

Comment:
This overrides the setting two lines up. Better modify the original one...

I think the original setting is better. The alphabetical index layout should be quite tight because it may contain a huge number of entries. We tested and found that 16px could be a good value.